### PR TITLE
Calcurse: update to 4.3.0

### PR DESCRIPTION
**Pull request by mklein-de**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G1611
Xcode 8.2.1 8C1002

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- x ] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/calcurse/Portfile [hunk 1]
```diff
@@ -1,8 +1,7 @@
 PortSystem 1.0
 
 name            calcurse
-version         3.1.4
-revision        1
+version         4.3.0
 categories      sysutils
 platforms       darwin
 maintainers     puffin.lb.shuttle.de:michael.klein openmaintainer
```
Comment (mojca) [inline sysutils/calcurse/Portfile]: Please use
```
maintainers     {puffin.lb.shuttle.de:michael.klein @mklein-de} openmaintainer
```
while at it.

Comment (mklein-de) [reply sysutils/calcurse/Portfile]: done

**Comment by mojca** [comment]
Thanks.

The builds failed on [10.8](https://build.macports.org/builders/ports-10.8_x86_64_legacy-builder/builds/43969) and [10.9](https://build.macports.org/builders/ports-10.9_x86_64-builder/builds/42548) with

```
clang: error: unknown argument: '-R/opt/local/lib'
```

**Comment by mklein-de** [comment]
#1051 should fix the build errors

**Pull request merged**